app/models/katello/content_view_docker_filter.rb

```diff
@@ -32,7 +32,7 @@ def fetch_tag_uuids(repo, rule, manifest_klass = DockerManifest)
 
       query = manifest_klass.in_repositories(repo).where("id in (#{tags_query.to_sql})")
       names = query.all.collect do |manifest|
-        manifest.docker_tags.where(:repository => repo).all.collect do |tag|
+        manifest.docker_tags.where(:repository => repo).where("name ilike ?", query_name).all.collect do |tag|
```

Comment:
can you add a line that does the following
```
query_name = query_name.name.tr("*", "%")
```

Reply:
@parthaa - I think that is already there?
https://github.com/Katello/katello/pull/7464/files#diff-c5101bb5147bb92312d8f0a90c94ead5R27

Reply:
good point.